CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ https://docs.docker.com/engine/deprecated/ where you can find the target removal
 * Update containerd/continuity to fix ARM 32-bit builds [moby/moby#36339](https://github.com/moby/moby/pull/36339)
```

Comment:
Not a user-facing change; safe to omit (as long as we're able to build those packages, right? 😄)

Reply:
The buildmode=pie is likely not interesting as well